### DISCUSSION TITLE
Welcome to Partner-Center-PowerShell Discussions!

### DISCUSSION THREAD

**Discussion by joelst** [Microsoft/Partner-Center-PowerShell#345] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Agazoth** [comment]
Hi,

My name is Axel Andersen. I have been in IT for 20+. I started out in PC management and continued in infrastructure architecture. The last 5 years I've been employed as Cloud Architect focused on Azure in various companies working as a DevOps engineer automating anything cloud.

My weapon of choice is PowerShell and I have been automating countless product integrations with it since version 1.0.

I love to learn and I like to contribute as much as time and abilities allow.

Working with Microsoft CSP I have found the Partner-Center-PowerShell cmdlets most useful. CSP is a rather small niche, so I'm really looking forward to having a community where ideas can be discussed and we can learn from each other.
